Prepare 16.04

greatfireball opened the pull request:
The installation procedure should work for Ubuntu 16.04 release.

greatfireball committed:
Deleted biber from package list

The installation problems caused by the explicit installation of the biber package should be solved for Ubuntu 16.04 now.

Fixes #5

greatfireball committed:
Installation of owncloud-client from standard repo

I added some code to distinguish between 16.04 Ubuntu and older distributions.

In case of a 16.04 owncloud client would be installed from the standard repository, in case of older releases it will be installed from the opensuse repository.

greatfireball committed:
Update 32_Owncloud.sh

Added a comment, if installing via opensuse repository

greatfireball committed:
Merge pull request #12 from cctb-it-wuerzburg/greatfireball-owncloud-installation-in-16.04

Installation owncloud-client via standard repo

greatfireball committed:
Dropbox installation with current package

The current dropbox installation package is now determined from the Dropbox download page (https://www.dropbox.com/install?os=lnx).

If more than one package might be retrieved, a message will be printed and no package will be installed.

greatfireball committed:
Remove guest login only for older Ubuntu releases

For Ubuntu 16.04 the disabling step for the guest login is not required any more.

For older releases the deactivation is still working

greatfireball committed:
Merge pull request #14 from cctb-it-wuerzburg/greatfireball-diable-guest-login

Remove guest login only for older Ubuntu releases

greatfireball committed:
R-repositories are added based on Ubuntu release

The r-pepository is now set according to the installed Ubuntu release.

greatfireball committed:
Merge pull request #15 from cctb-it-wuerzburg/greatfireball-r-repo-bases-on-current-release

R-repositories are added based on Ubuntu release

greatfireball committed:
Deleted Ubuntu version from file name

I deleted the Ubuntu version number from the filename and updated the text to point to https://github.com/greatfireball/create-ubuntu-autoinstall-medium for an automated installation of Ubuntu.

Fixes #10

review-ninja commented:
<a href="https://app.review.ninja/cctb-it-wuerzburg/installation/pull/16" target="_blank"><img src="https://app.review.ninja/55073444/pull/16/badge" alt="ReviewNinja"/>

greatfireball committed:
Merge branch 'master' into prepare_16.04

greatfireball committed:
Deleted installation of openjdk-7-jre

The package openjdk-7-jre was deleted from the list of additional packages

Fixes #17

greatfireball committed:
Deleted some codecs, which caused problems

The codecs:

```
libxine1-ffmpeg
libmpeg3-1
libflac++6
totem-mozilla
libswscale-extra-2
```
have been removed from additional.packages.list
